hbase-server/src/main/java/org/apache/hadoop/hbase/io/asyncfs/FanOutOneBlockAsyncDFSOutputHelper.java

```diff
@@ -275,10 +275,9 @@ public boolean progress() {
       DFS_CLIENT_ADAPTOR = createDFSClientAdaptor();
       FILE_CREATOR = createFileCreator();
     } catch (Exception e) {
-      String msg = "Couldn't properly initialize access to HDFS internals. Please " +
-          "update your WAL Provider to not make use of the 'asyncfs' provider. See " +
+      String msg = "Failed to properly initialize access to HDFS internals. Please " +
```

Comment:
what does the log look like when this failure bubbles up the call stack? (I would like an answer before this merges)